src/robotide/ui/tagdialogs.py

```diff
@@ -100,6 +101,14 @@ def _make_bindings(self):
         self._tags_list.Bind(wx.EVT_LIST_ITEM_RIGHT_CLICK, self.OnRightClick)
         self._tags_list.Bind(wx.EVT_LIST_COL_CLICK, self.OnColClick)
 
+    def _tag_name_for_sort(self, tag_name):
+        # Make tag_name lowercase for the sorting algorithm only
+        tag_name = tag_name.lower()
+        # Compare trailing numbers as numeric values
+        m = re.search(r'([^0-9]*)([0-9]*)', tag_name)
```

Comment:
According to the comment this regexp tries to to find trailing numbers, but the regexp itself actually finds *any* number. Is comment or code correct??